### ISSUE TITLE
Dashboard - creditor name missing

### ISSUE THREAD

**Issue by magnolia61** [Project60/sepa_dd#227]
Right now the dashboard groups mandates by OOFF / RCUR and a different group per creditor ID.
From the dashboard it is not visible which TX group belongs to which creditor. Can the creditor name be included as a colomn on the dashboard screen and listed on the listgroup screen as well.

It will clarify and help easy management of multiple creditors and groups derived from donations, events and memberships.

![sepadd_dashboard_missingcreditor](https://cloud.githubusercontent.com/assets/2195908/5711567/d61b63c4-9aab-11e4-9437-c6a85542c0ba.png)

**Comment by systopia** [comment]
> From the dashboard it is not visible which TX group belongs to which creditor. 

The creditor ID is the number that follows the ``TXG-`` in the group name. I admit it's not nice, but at least you have the information.

In the future, we might want to include the creditor name, but I'm thinking more along the lines of a hover value rather than an extra column. Keep in mind that not everybody has as much "screen real estate" as you :)

**Comment by antrik** [comment]
On Wed, Jan 14, 2015 at 02:13:57AM -0800, SYSTOPIA wrote:
> In the future, we might want to include the creditor name, but I'm thinking more along the lines of a hover value rather than an extra column.
[...]
> Keep in mind that not everybody has as much "screen real estate" as you :) 

Note though that small display devices usually do not have a "hover" capability either...

**Comment by systopia** [comment]
> Note though that small display devices usually do not have a "hover" capability either...

Good point. However, on those devices the screen real estate problem is even more severe. Any better ideas?

**Comment by magnolia61** [comment]
Maybe use the mandate_prefix in the groupname? Would only cost 4 characters?

**Comment by systopia** [comment]
> Maybe use the mandate_prefix in the groupname? Would only cost 4 characters?

In our scenarios the mandate prefix would be the same for most creditors, as it's mostly related to the organisation's name.

**Comment by antrik** [comment]
Not sure that helps; but in the SfE variant, all dashboard entries are grouped by Creditor at the top level. I think it's clearer that way -- and we don't need to include the Creditor name in the individual entries...
